### ISSUE TITLE
General statements 

### ISSUE THREAD

**Issue by njkeng** [almossawi/badarguments#11]
On page 6 of the book the text reads "and had things like, “try not to make general claims about things.” That is obvious to me now .. ". The issue is that the statement "try not to make general claims about things " is in itself a general claim. It speaks about how arguments should be conducted. Perhaps making general claims is not so bad after all? Perhaps some general statements are not as obvious as others.

**Comment by almossawi** [comment]
You're right. I suppose adding "without evidence" to the end of that line would avoid the ambiguity.

**Comment by almossawi** [comment]
Done and deployed, credited you [here](https://github.com/almossawi/badarguments/wiki/Contributors). 👍
